src/soundio/sounddeviceportaudio.cpp

```diff
@@ -211,25 +212,26 @@ SoundDeviceError SoundDevicePortAudio::open(bool isClkRefDevice, int syncBuffers
         m_dSampleRate = 44100.0;
     }
 
-    // Get latency in milleseconds
-    qDebug() << "framesPerBuffer:" << m_framesPerBuffer;
-    double bufferMSec = m_framesPerBuffer / m_dSampleRate * 1000;
-    qDebug() << "Requested sample rate: " << m_dSampleRate << "Hz, latency:"
+    SINT framesPerBuffer = m_configFramesPerBuffer;
+    if (m_deviceTypeId == paJACK) {
+        // PortAudio's JACK back end has its own buffering to split or merge the buffer
+        // received from JACK to the desired size.
+        // However, we use here paFramesPerBufferUnspecified to use the JACK buffer size
+        // which offers the best responds time without additional jitter.
+        framesPerBuffer = paFramesPerBufferUnspecified;
+        qDebug() << "framesPerBuffer: Unspecified";
```

Comment:
I suspect this log line will confuse users. I suggest simply deleting it, or rewording it to something like `framesPerBuffer: Using JACK server's frames per buffer`